core/src/main/resources/defaults/blueprints/cdp-sdx.bp

```diff
@@ -259,101 +295,53 @@
             "value": "1"
           }
         ],
+        "serviceType": "KAFKA"
+      },
+      {
+        "refName": "knox",
+        "serviceType": "KNOX",
         "roleConfigGroups": [
           {
-            "refName": "kafka-GATEWAY-BASE",
-            "roleType": "GATEWAY",
-            "base": true
-          },
-          {
-            "refName": "kafka-KAFKA_BROKER-BASE",
-            "roleType": "KAFKA_BROKER",
+            "refName": "knox-KNOX_GATEWAY-BASE",
+            "roleType": "KNOX_GATEWAY",
+            "configs": [
+              {
+                "name": "ldap_connection_password",
+                "value": "admin123!"
+              },
+              {
+                "name": "ldap_connection_url",
+                "value": "ldap://ipa.vpc.cloudera.com:389"
+              },
+              {
+                "name": "gateway_master_secret",
+                "value": "master_secret"
```

Comment:
I am not sure if this will be ultimately configurable in CB during cluster provisioning, but having it hardwired in the template is probably not advisable.

Reply:
will remove the LDAP configurations.